radix_sort/Java/radix_sort.java

```diff
@@ -0,0 +1,27 @@
+public class RadixSort {
+	public static void sort( int[] a) {
+        int i, m = a[0], exp = 1, n = a.length;
+        int[] b = new int[n];
+        for (i = 1; i < n; i++)
+            if (a[i] > m)
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp16hx5pms/radix_sort/Java/radix_sort.java
+++ b/tmp/tmp16hx5pms/radix_sort/Java/radix_sort.java
@@ -1,5 +1,5 @@
 public class RadixSort {
-	public static void sort( int[] a) {
+    public static void sort( int[] a) {
         int i, m = a[0], exp = 1, n = a.length;
         int[] b = new int[n];
         for (i = 1; i < n; i++)
```